src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,682 @@
+
+```js title="astro.config.mjs"
+weights: [200, "300", "bold"]
+```
+
+If the associated font is a [variable font](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_fonts/Variable_fonts_guide), you can specify the weights range:
+
+```js title="astro.config.mjs"
+weights: ["100 900"]
+```
+
+##### styles
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `["normal", "italic"]`
+</p>
+
+An array of [font styles](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style):
+
+```js title="astro.config.mjs"
+styles: ["normal", "oblique"]
+```
+
+##### subsets
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `["cyrillic-ext", "cyrillic", "greek-ext", "greek", "vietnamese", "latin-ext", "latin"]`
+</p>
+
+An array of [font subsets](https://fonts.google.com/knowledge/glossary/subsetting):
+
+```js title="astro.config.mjs"
+subsets: ["latin"]
+```
+
+##### display
+
+<p>
+
+**Type:** `string`<br />
+**Default:** `"swap"`
+</p>
+
+An [font display](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display):
+
+```js title="astro.config.mjs"
+display: "block"
+```
+
+##### unicodeRange
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `undefined`
+</p>
+
+A [unicode range](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range):
+
+```js title="astro.config.mjs"
+unicodeRange: ["U+26"]
+```
+
+##### stretch
+
+<p>
+
+**Type:** `string`<br />
+**Default:** `undefined`
+</p>
+
+A [font stretch](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-stretch):
+
+```js title="astro.config.mjs"
+stretch: "condensed"
+```
+
+##### featureSettings
+
+<p>
+
+**Type:** `string`<br />
+**Default:** `undefined`
+</p>
+
+Font [feature settings](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-feature-settings):
+
+```js title="astro.config.mjs"
+featureSettings: "'smcp' 2"
+```
+
+##### variationSettings
+
+<p>
+
+**Type:** `string`<br />
+**Default:** `undefined`
+</p>
+
+Font [variation settings](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-variation-settings):
+
+```js title="astro.config.mjs"
+variationSettings: "'xhgt' 0.7"
+```
+
+#### Available remote font providers
+
+Astro reexports most [unifont](https://github.com/unjs/unifont/) providers:
+
+```js
+import { fontProviders } from "astro/config"
+```
+
+##### adobe
+
+```js title="astro.config.mjs"
+provider: fontProviders.adobe({
+    id: process.env.ADOBE_ID
+})
+```
+
+##### bunny
+
+```js title="astro.config.mjs"
+provider: fontProviders.bunny()
+```
+
+##### fontshare
+
+```js title="astro.config.mjs"
+provider: fontProviders.fontshare()
+```
+
+##### fontsource
+
+```js title="astro.config.mjs"
+provider: fontProviders.fontsource()
+```
+
+##### google
+
+```js title="astro.config.mjs"
+provider: fontProviders.google()
+```
+
+##### Custom providers
+
+Any unifont provider can be used but it cannot be as is. [Learn how to make a custom Astro font provider](#font-provider).
```

Comment:
```suggestion
Any unifont provider can be used but may require additional setup. [Learn how to make a custom Astro font provider](#font-provider).
```

Not sure if this is the correct fix here but I think it's a bit clearer

Reply:
No what I mean is literally you can't do `provider: someUnifontProvider()`. You have to use our API, which works with unifont under the hood

Reply:
Ah okay, now I understand

I do think it needs to be clarified a bit still but might require one of our docs experts haha

Maybe just a simple `The Astro Fonts API supports any unifont provider`

I think this followed by the link text we already have makes it a bit more clear, however like I said could maybe still be improved